### ISSUE TITLE
Feature Request: Support Different File Sorting

### ISSUE THREAD

**Issue by brendon-felix** [FabianUntermoser/obsidian-neighbouring-files-plugin#13]
Would be nice if there was a setting or new set of commands to switch the sort order, ideally based on the file explorer sort setting. Instead of just alphabetical order, neighboring file could be the previously created/modified and  could switch between ascending and descending.

**Comment by FabianUntermoser** [comment]
Good idea, I agree. Won't be able to work on this until next year though. Feel free to add a PR 👍

**Comment by Cr8zyIvan** [comment]
I support this feature request 😊

**Comment by FabianUntermoser** [comment]
Sorting by creation and modificaiton timestamp now available in [Release 1.0.7](https://github.com/FabianUntermoser/obsidian-neighbouring-files-plugin/releases/tag/1.0.7)

See #15 for more details

**Comment by brendon-felix** [comment]
Just added functionality for the standard commands to follow file explorer sort order.
See #17
